fiat-api/fiat-api.gradle

```diff
@@ -15,33 +15,22 @@
  */
 
 dependencies {
-  compile project(":fiat-core")
+  implementation project(":fiat-core")
 
-  compileOnly spinnaker.dependency("bootWeb")
-  compileOnly spinnaker.dependency("frigga")
-  compileOnly spinnaker.dependency("korkSecurity")
-  compileOnly spinnaker.dependency("korkWeb")
-  compileOnly spinnaker.dependency("kork")
-  compileOnly spinnaker.dependency("lombok")
-  annotationProcessor spinnaker.dependency("lombok")
-  compileOnly spinnaker.dependency("okHttp")
-  compileOnly spinnaker.dependency("okHttpUrlconnection")
-  compileOnly spinnaker.dependency("okHttpApache")
-  compileOnly spinnaker.dependency("retrofit")
-  compileOnly spinnaker.dependency("retrofitJackson")
-  compileOnly spinnaker.dependency("spectatorApi")
+  implementation "org.springframework:spring-aop"
```

Comment:
these all are actually runtime dependencies. With BOM based dependency opinions we won't run into problems on version alignment so it is safe to make these implementation rather than compileOnly